deploy/charts/vault-secrets-reloader/values.yaml

```diff
@@ -4,6 +4,8 @@
 
 # -- Log level
 logLevel: info
+# -- Use JSON log format instead of text
+enableJSONLog: false
```

Comment:
Consider using `logFormat` with allowed values of `text` and `json`. 
You can silently fallback to `text` in the CLI iff the value is invalid/unsupported.

Reply:
That was my initial plan as well but then I got sidetracked by [this](https://github.com/bank-vaults/vault-secrets-webhook/blob/433983d79d88a3e1e602ea1cf2551992786424eb/pkg/webhook/config.go#L291)... gonna change it back as you suggested 🙂

Reply:
In that case, feel free to keep it, no need to further complicate it.
And also, text + json should suffice for options.